course_info/templates/course_info/partials/details.html

```diff
@@ -6,208 +6,256 @@ <h3 class="breadcrumbs">
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Meeting Time: </strong>
-                    </div>
-                    <div class="col-md-10">{{dc.courseInstance.meeting_time}}</div>
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Departments: </strong>
-                    </div>
-                    <div class="col-md-10" ng-repeat="dept in dc.courseInstance.departments">
-                         <div class="list-group" ng-if="dc.courseInstance.departments.length > 0">
-                            <div class="list-group-item list-group-sub-item" ng-repeat="dept in dc.courseInstance.departments">
-                                {{dept.name}}
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Course Groups: </strong>
-                    </div>
-                    <div class="col-md-10">
-                        <div class="list-group">
-                            <div class="list-group-item list-group-sub-item" ng-repeat="group in dc.courseInstance.course_groups">
-                                {{group.name}}
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </li>
-
-            <li class="list-group-item">
-                <div class="row">
-                    <div class="col-md-2">
-                        <strong>Associated Official and Unofficial Site(s): </strong>
-                    </div>
-                    <div class="col-md-10">
-                        <div class="list-group">
-                            <div class="list-group-item list-group-sub-item" ng-repeat="site in dc.courseInstance.sites">
-                                <div ng-switch on="site.map_type">
-                                    <div class="label label-primary col-md-1" ng-switch-when="official">Official</div>
-                                    <div class="label label-primary col-md-1" ng-switch-when="unofficial">Unofficial</div>
-                                </div>
-                                <div class="col-md-11"><a href="{{site.course_site_url}}"
-                                                          target="_blank">{{site.course_site_url}}</a>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+            <hu-editable-input
+                is-loading="dc.isDefined(dc.courseInstance.registrar_code_display)"
```

Comment:
FYI I cannot get the default to `angular.isUndefined()` to work -- it requires some combination of accessing the `attrs` and isolate `$scope` via the compile, link, and controller functions for the directive that I cannot quite get right. I'm going to skip this suggestion for the purposes of getting out working code, and we can refactor in parallel with the jasmine tests. I'd be very happy to see these directives improved. One way, as mentioned, is simply to make them use the parent scope, instead of isolate scopes -- then you'd really only have to pass in the `label` and the `field`, and possibly the `editable` and an `isLoading` override -- everything else could be calculated/deduced from the name of the field.